### PR TITLE
updated librispeech-asr with rnn-t results

### PR DESCRIPTION
**Pull request by chintu619**

### CODE REVIEW

**Comment by chintu619** [comment]
Initially, I only did inference on test sets. After I do inference on dev sets, I will update the readme file again.

File: egs2/librispeech/asr1/README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+# RNN-T Conformer
```
Comment (b-flo) [inline egs2/librispeech/asr1/README.md:1]:
```suggestion
# Conformer-RNN Transducer
```

File: egs2/librispeech/asr1/README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+
+## Environments
+- date: `Fri Mar 25 04:35:42 EDT 2022`
+- python version: `3.9.5 (default, Jun  4 2021, 12:28:51)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.7a1`
+- pytorch version: `pytorch 1.8.1+cu111`
+- Git hash: `21d19be00089678ca27f7fce474ef8d787689512`
+  - Commit date: `Wed Mar 16 08:06:52 2022 -0400`
+- ASR config: [conf/train_rnnt_conformer.yaml](conf/train_rnnt_conformer.yaml)
```
Comment (b-flo) [inline egs2/librispeech/asr1/README.md:10]:
```suggestion
- ASR config: [conf/tuning/transducer/train_conformer-rnn_transducer.yaml](conf/train_rnnt_conformer.yaml)
```

**Review by b-flo — COMMENTED** [review]
Thanks a lot for working on a Transducer recipe for Librispeech!
The changes I propose are only for consistency, I/we usually name transducer configs and models with the following patterns, respectively: `[encoder]-[decoder]_transducer` and `[encoder]-[decoder] Transducer`.

If it's possible and you have some bandwidth available, could you try training the same model with #4032 version please?

File: egs2/librispeech/asr1/README.md (hunk 2 above)
Comment (chintu619) [reply egs2/librispeech/asr1/README.md:10]: Thank you for the great suggestions. Also, I will try to train the model from #4032 when my cluster frees up.

**Pull request merged**